tests/test_output/tmva_net_column_totals.out

```diff
@@ -109,176 +112,181 @@ TMVANet                                       [1, 4, 256, 64]           --
 │    │    │    │    └─Conv3d: 5-4             [1, 128, 1, 256, 256]     442,496                   28,999,417,856
 │    │    │    │    └─BatchNorm3d: 5-5        [1, 128, 1, 256, 256]     256                       256
 │    │    │    │    └─LeakyReLU: 5-6          [1, 128, 1, 256, 256]     --                        --
-│    │    └─MaxPool2d: 3-2                    [1, 128, 128, 128]        --                        --
-│    │    └─DoubleConvBlock: 3-3              [1, 128, 128, 128]        --                        --
+│    │    └─MaxPool2d: 3-2                    --                        --                        --
```

Comment:
Are these being duplicated?

Reply:
Are you referring to `MaxPool2d` layers that appear consecutively in `EncodingBranch`?
If so, they stem from the two kinds of the following max pools layers defined in `EncodingBranch`

```
self.doppler_max_pool = nn.MaxPool2d(2, stride=(2, 1))
self.max_pool = nn.MaxPool2d(2, stride=2)
```
Since one of them is only defined (not used), it appears once. The other is called twice, and it appears twice.